packages/core/services/google-maps-types.ts

```diff
@@ -82,6 +82,13 @@ export interface MarkerLabel {
   text: string;
 }
 
+export interface MarkerIcon {
+  url: string;
+  scaledSize?: any | Size;
+  origin?: any | Point;
+  anchor?: any | Point;
```

Comment:
what about `labelOrigin` and `size`? Also, they have to be in alphabetical order

Reply:
Got it, I've added it according to Google Map Doc

https://developers.google.com/maps/documentation/javascript/reference/marker#Icon